wordpress-mobile/WordPress-iOS
# Fixes Podfile.lock out of sync on trunk

Fixes #NA

To test:

- checkout this branch
- `rake dependencies`
-  make sure the app builds

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Build the app on Xcode

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
